<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INSML-H55MD6M" failures="1" tests="11" name="com.paraBanking.TestRunner.TestRunner" time="538.980" errors="1" timestamp="2022-12-26T23:09:17 IST" skipped="0">
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="38.910"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="38.690"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="37.478"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="37.220"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="38.531"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="39.280"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="135.182">
    <error message="Expected condition failed: waiting for com.paraBanking.stepDefinitions.ParaBankStepDefinitions$$Lambda$728/0x00000008004e93c0@542adcfa (tried for 120 second(s) with 5000 milliseconds interval)" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for com.paraBanking.stepDefinitions.ParaBankStepDefinitions$$Lambda$728/0x00000008004e93c0@542adcfa (tried for 120 second(s) with 5000 milliseconds interval)
at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:263)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)
at com.paraBanking.stepDefinitions.ParaBankStepDefinitions.openAccount(ParaBankStepDefinitions.java:204)
at ✽.Open New Savings Account(file:///Users/snimma868/Demo/paraBanking/src/test/resources/paraBanking.feature:31)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//h1[text()="Account Opened!"]"}
  (Session info: chrome=108.0.5359.124)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.6.0', revision: '79f1c02ae20'
System info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '12.0.1', java.version: '17.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [2c3d9ad7898bbf037593ab9d9a6befb3, findElement {using=xpath, value=//h1[text()="Account Opened!"]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 108.0.5359.124, chrome: {chromedriverVersion: 108.0.5359.71 (1e0e3868ee06..., userDataDir: /var/folders/fr/zhkkvj710pv...}, goog:chromeOptions: {debuggerAddress: localhost:51548}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: MAC, proxy: Proxy(), se:cdp: ws://localhost:51548/devtoo..., se:cdpVersion: 108.0.5359.124, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 2c3d9ad7898bbf037593ab9d9a6befb3
at jdk.internal.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:541)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:365)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:357)
at com.paraBanking.stepDefinitions.ParaBankStepDefinitions.lambda$0(ParaBankStepDefinitions.java:205)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
at com.paraBanking.stepDefinitions.ParaBankStepDefinitions.openAccount(ParaBankStepDefinitions.java:204)
at jdk.internal.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at io.cucumber.java.Invoker.doInvoke(Invoker.java:66)
at io.cucumber.java.Invoker.invoke(Invoker.java:24)
at io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)
at io.cucumber.java.JavaStepDefinition.execute(JavaStepDefinition.java:29)
at io.cucumber.core.runner.CoreStepDefinition.execute(CoreStepDefinition.java:66)
at io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:63)
at io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)
at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:85)
at io.cucumber.core.runner.TestStep.run(TestStep.java:57)
at io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)
at io.cucumber.core.runner.TestCase.run(TestCase.java:84)
at io.cucumber.core.runner.Runner.runPickle(Runner.java:75)
at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)
at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:129)
at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:129)
at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="38.257"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="33.814"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="38.119"/>
  <system-out/>
  <testcase classname="com.paraBanking.TestRunner.TestRunner" name="runScenario" time="63.499">
    <failure message="Amount didn&#039;t get credited to Riya" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Amount didn't get credited to Riya
at org.testng.Assert.fail(Assert.java:110)
at com.paraBanking.stepDefinitions.ParaBankStepDefinitions.verifyBalanceAndTransactionTable(ParaBankStepDefinitions.java:423)
at ✽.Verify that balance and transaction table details of Checking account is correct after receiving amount of 50 from Riya(file:///Users/snimma868/Demo/paraBanking/src/test/resources/paraBanking.feature:23)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- com.paraBanking.TestRunner.TestRunner -->
